renovate/devcontainer.json

```diff
@@ -8,5 +8,13 @@
         "//\\s*renovate: datasource=(?<datasource>.+?)\\s+depName=(?<depName>.+?)\\s+\".*[Vv]ersion\":\\s*\"(?<currentValue>.+?)\""
       ]
     }
+  ],
+  "packageRules": [
+    {
+      "matchManagers": ["dockerfile"],
+      "matchFileNames": [".devcontainer/Dockerfile"],
```

Comment:
oh neat, I guess I hadn't migrated this to `Containerfile`

Reply:
I'll add that path too!